app/assets/javascripts/application.js

```diff
@@ -1,107 +1,121 @@
 //= require jquery
 //= require textinputs_jquery
 //= require jquery-cookie/jquery.cookie
+//= require three.min
+//= require fort-scene
 
-function anim_show(el) {
-  $(el).fadeIn();
-}
+(function() {
+  function anim_show(el) {
+    $(el).fadeIn();
+  }
 
-function anim_hide(el) {
-  $(el).fadeOut();
-}
+  function anim_hide(el) {
+    $(el).fadeOut();
+  }
 
-function insert_emoticon(face) {
-  var my_text = $('#leaf_content').val();
-  var caret_pos = $('#leaf_content').getSelection().end;
+  function insert_emoticon(face) {
+    var my_text = $('#leaf_content').val();
+    var caret_pos = $('#leaf_content').getSelection().end;
 
-  $('#leaf_content').val(
-    my_text.substring(0, caret_pos)
-    + face
-    + my_text.substring(caret_pos)
-    );
+    $('#leaf_content').val(
+      my_text.substring(0, caret_pos)
+      + face
+      + my_text.substring(caret_pos)
```

Comment:
Identifier 'my_text' is not in camel case.<br>Bad line breaking before '+'.<br>Identifier 'caret_pos' is not in camel case.